src/components/WalletDropdown/MiniPortfolio/PortfolioLogo.tsx

```diff
@@ -147,12 +136,10 @@ export function PortfolioLogo({
   }
 
   const L2Logo =
-    chainId === SupportedChainId.MAINNET ? null : (
-      <div>
-        {chainLogo && <ChainLogoSquareBackground />}
-        {!squareLogoUrl && logoUrl && <SquareBackgroundForNonSquareLogo />}
-        {chainLogo && <StyledChainLogo isSquare={!!squareLogoUrl} src={chainLogo} alt="chainLogo" />}
-      </div>
+    chainId === SupportedChainId.MAINNET || !chainLogo ? null : (
+      <L2LogoContainer nonSquareLogo={Boolean(!squareLogoUrl && logoUrl)}>
```

Comment:
in general im not a fan of dispersing the custom logic here within two components - so there's some logic on the `PortfolioLogo` level to determine the logo to be used and also `isSquare` logic, and then there's custom logic within `L2LogoContainer` on the background-color 

feels like they should all be prepared within just the `PortfolioLogo` container by determining the background color here instead of within `L2LogoContainer`

that way you also avoid passing in a non-native prop to an html element and also take out logic from css